### PR TITLE
Js code refactor formatting and removal of excess files

### PR DESCRIPTION
**Pull request by kaylorburdell**
@dm-istomin @beatrixbetsycarroll 

we have cleaned up the code a bit

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV["RAILS_ENV"] ||= 'test'
-require 'spec_helper'
+
```
Comment (dm-istomin) [inline spec/rails_helper.rb]: @matthewhuff89 @kaylorburdell A little wary of deleting requires from Rails files. I'd put it back for now.

File: spec/javascripts/support/jasmine_helper.rb [hunk 2]
```diff
@@ -1,15 +0,0 @@
-#Use this file to set/override Jasmine configuration options
```
Comment (dm-istomin) [inline spec/javascripts/support/jasmine_helper.rb]: @matthewhuff89 @kaylorburdell - Not sure about deleting this, is it gonna interfere with us running Jasmine tests in the future? We still want to test our JavaScript.

**Comment by dm-istomin** [comment]
@matthewhuff89 @kaylorburdell Looks good, merging it in.

File: app/views/cards/_card.html.erb [hunk 3]
```diff
@@ -1,10 +1,12 @@
 <div class="card">
-  <span class='name'><h2><%= @card.name %></h2></span>
-  <div class = "card_pic">
-    <img src="<%= @card.picture_url %>", class="portrait">
-  </div>
+    <span class='name'><h2><%= @card.name %></h2></span>
```
Comment (sgharms) [inline app/views/cards/_card.html.erb]: Indent out

File: app/views/cards/index.html.erb [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <ol>
-<% @cards.each do |card| %>
-<h3><%= link_to card.name, card_path(card) %></h3>
-<% end %>
+  <% @cards.each do |card| %>
+    <h3><%= link_to card.name, card_path(card) %></h3>
```
Comment (sgharms) [inline app/views/cards/index.html.erb]: Indent out

File: spec/game/game_runner_spec.rb [hunk 5]
```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
-require 'spec_helper'
+
```
Comment (sgharms) [inline spec/game/game_runner_spec.rb]: I think you have a trailing newline here. It should have just been s red delete with nothing added

**Comment by sgharms** [comment]
Good work cleaning

**Pull request merged**